tools/coverage/docs/runLocally.md

```diff
@@ -0,0 +1,15 @@
+A pre-submit check, to see how the coverage of your local repository compare to the lastest postsubmit run, can be run locally in the following way
```

Comment:
To keep the file naming convention, rename this to `run_local.md`, `running_locally.md`, etc.